src/bootstrap/src/core/build_steps/doc.rs

```diff
@@ -834,19 +830,12 @@ fn doc_std(
         }
     }
 
-<<<<<<< HEAD
     // Ferrocene addition
     if target.contains("ferrocene.certified") {
         cargo.rustdocflag("--cfg=ferrocene_certified");
         cargo.arg("--features").arg("ferrocene_certified");
     }
 
-    if builder.config.library_docs_private_items {
-        cargo.rustdocflag("--document-private-items").rustdocflag("--document-hidden-items");
-    }
```

Comment:
my git skills fail me... how did this go away

Reply:
I had not realized that this is a ferrocene-specific option. I don't think it needs to be? I'll chat with upstream and see if they want it.

In the meantime I'll read-add this config and add a comment.